modules/openapi-generator/src/main/resources/csharp-netcore/libraries/generichost/JsonConverter.mustache

```diff
@@ -213,8 +213,27 @@
         /// <exception cref="NotImplementedException"></exception>
         public override void Write(Utf8JsonWriter writer, {{classname}} {{#lambda.camelcase_param}}{{classname}}{{/lambda.camelcase_param}}, JsonSerializerOptions jsonSerializerOptions)
         {
+            {{#composedSchemas.anyOf}}
+            {{^vendorExtensions.x-duplicated-data-type}}
+            System.Text.Json.JsonSerializer.Serialize(writer, {{#lambda.camelcase_param}}{{classname}}{{/lambda.camelcase_param}}.{{#lambda.titlecase}}{{baseType}}{{#isArray}}{{{dataFormat}}}{{/isArray}}{{/lambda.titlecase}}, jsonSerializerOptions);
+
+            {{/vendorExtensions.x-duplicated-data-type}}
+            {{/composedSchemas.anyOf}}
+            {{#composedSchemas.oneOf}}
+            {{^vendorExtensions.x-duplicated-data-type}}
+            System.Text.Json.JsonSerializer.Serialize(writer, {{#lambda.camelcase_param}}{{classname}}{{/lambda.camelcase_param}}.{{#lambda.titlecase}}{{baseType}}{{#isArray}}{{{dataFormat}}}{{/isArray}}{{/lambda.titlecase}}, jsonSerializerOptions);
+
+            {{/vendorExtensions.x-duplicated-data-type}}
+            {{/composedSchemas.oneOf}}
+            {{#composedSchemas.allOf}}
+            {{^isInherited}}
+            System.Text.Json.JsonSerializer.Serialize(writer, {{#lambda.camelcase_param}}{{classname}}{{/lambda.camelcase_param}}.{{#lambda.titlecase}}{{baseType}}{{#isArray}}{{{dataFormat}}}{{/isArray}}{{/lambda.titlecase}}, jsonSerializerOptions);
+
+            {{^composedSchemas}}
             writer.WriteStartObject();
 
+            {{/isInherited}}
+            {{/composedSchemas.allOf}}
```

Comment:
looks like these 2 lines should move up to line 231 as currently we get 
```
[pool-1-thread-2] Generation failed for csharp-netcore: (RuntimeException) Could not generate model 'apple'
java.lang.RuntimeException: Could not generate model 'apple'
	at org.openapitools.codegen.DefaultGenerator.generateModels(DefaultGenerator.java:569)
	at org.openapitools.codegen.DefaultGenerator.generate(DefaultGenerator.java:926)
	at org.openapitools.codegen.cmd.GenerateBatch$GenerationRunner.run(GenerateBatch.java:232)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: com.samskivert.mustache.MustacheParseException: Section close tag with mismatched open tag 'isInherited' != 'composedSchemas' @ line 235
	at com.samskivert.mustache.Mustache$Accumulator.requireSameName(Mustache.java:754)
	at com.samskivert.mustache.Mustache$Accumulator$2.addCloseSectionSegment(Mustache.java:705)
	at com.samskivert.mustache.Mustache$Accumulator.addTagSegment(Mustache.java:713)
	at com.samskivert.mustache.Mustache$Parser.parseChar(Mustache.java:573)
	at com.samskivert.mustache.Mustache$Parser.parse(Mustache.java:471)

```

Reply:
filed https://github.com/OpenAPITools/openapi-generator/pull/15424 to fix/workaround it for the time being.

please take another look when you've time.